src/browser/web-card.js

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
+
+import {Record, Union, List, Maybe, Any} from 'typed-immutable';
```

Comment:
> for ES Modules. Is this going to cause any problems with other modules that are AMD?

babel compiles everything into UMD so after compile step it's all fine. If you would to try and load this with requirejs it will fail because this syntax is not even implemented by a browsers. Although all other modules will also fail with requirejs because we no longer wrap them into `define(...)` as that was confusing bundler.